client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,17 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
+		return (
+			<EmptyContent
+				title={ this.props.translate( 'Not available for multi site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Domains are not available for this site.' )` *ES Score: 6.50*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/disable-upload-for-multisite)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).